docs-chef-io/content/inspec/resources/ipnat.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "ipnat resource"
+draft = false
+gh_repo = "inspec"
+platform = "bsd"
```

Comment:
A couple issues here: as mentioned by @IanMadd to https://github.com/inspec/inspec/pull/5880 PR

This platform parameter is used to sort resource pages in this page: https://docs.chef.io/inspec/resources/ and we don't currently have bsd or solaris platforms listed there (although we could add them)

The toml spec doesn't accept repeating entries.

It's possible that we could define this as a list of platforms, but I don't think that will work on the resource list page right now.

So I guess I have two questions:

Would this page fit under one of the existing headings on the [inspec/resources](https://docs.chef.io/inspec/resources/) page? (I'm guessing not)
Will there be a bunch more resources that are in bsd and/or solaris? In which case I can add headings for both and figure out how to handle these platforms in a list.